spring-test/src/test/java/org/springframework/test/context/jdbc/NonTransactionalSqlScriptsTests.java

```diff
@@ -42,7 +42,7 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @ContextConfiguration(classes = EmptyDatabaseConfig.class)
-@Sql({ "schema.sql", "data.sql" })
+@Sql({"drop-schema.sql", "schema.sql", "data.sql" })
```

Comment:
Why did you make this change?

As far as I recall, there is no need to drop the schema, since the database starts off empty.